torch/utils/tensorboard/beholder/__init__.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
We definitely need to remove the TensorBoard authors headers from all these files. I don't think they actually came from TensorFlow/Board, right?

Reply:
@orionr My beholder port has not been totally tested. I would like to remove them from integration as for now. These files are copied from tensorboard.